### ISSUE TITLE
ipywidgets not working

### ISSUE THREAD

**Issue by dominikstraessle** [jupyterlab/jupyterlab#3925]
I become the following error when executong this:

`import ipywidgets as widgets`
`widgets.Button(description='Hello')`

> Failed to display Jupyter Widget of type Button.
> 
> If you're reading this message in the Jupyter Notebook or JupyterLab Notebook, it may mean that the widgets JavaScript is still loading. If this message persists, it likely means that the widgets JavaScript library is either not installed or not enabled. See the Jupyter Widgets Documentation for setup instructions.
> 
> If you're reading this message in another frontend (for example, a static rendering on GitHub or NBViewer), it may mean that your frontend doesn't currently support widgets.

I enabled the extension before like this:
`jupyter nbextension enable --py widgetsnbextension`

In jupyter notebook the same code works without problems. It's only in jupyter lab.

The same error occured also after restarting the notebook/browser/system and in different browsers.

**Comment by dhirschfeld** [comment]
The notebook extension (`nbextension`) won't work for jupyterlab. The jupyterlab extension can be installed with:
```
jupyter labextension install @jupyter-widgets/jupyterlab-manager
```

***Ref:*** https://ipywidgets.readthedocs.io/en/latest/user_install.html#installing-the-jupyterlab-extension

**Comment by dominikstraessle** [comment]
Thank you!

**Comment by jasongrout** [comment]
What version of ipywidgets do you have, and what version of JupyterLab do you have?

**Comment by Gabrielvon** [comment]
@jasongrout 

Hi, please see below.

<img width="427" alt="image" src="https://user-images.githubusercontent.com/8952869/45831070-059ca800-bd31-11e8-9392-3ab6b7599de1.png">

**Comment by jasongrout** [comment]
Can you post the ipywidgets version in the kernel you are running? `import ipywidgets; ipywidgets.__version__`

**Comment by Gabrielvon** [comment]
here. It seems the same.

![image](https://user-images.githubusercontent.com/8952869/45855318-27724b00-bd82-11e8-9ff8-9dff6a8d53a2.png)

**Comment by jasongrout** [comment]
Let's start at the beginning, then, since the message in this issue cannot come from ipywidgets 7.4.1. We must have missed something. What exactly is the problem you are seeing?

**Comment by Gabrielvon** [comment]
Sure. 

Initially, I came across some problems using `tqdm_notebook` in jupyterlab. After googling the problems, someone say it may be something wrong with ipywidgets. They basically suggest that running `jupyter labextension install @jupyter-widgets/jupyterlab-manager`. So I did, but it doesn't work either for `tqdm_notebook` or`ipywidgets`. The following pic may present my problems.

![image](https://user-images.githubusercontent.com/8952869/45861258-26e7ad80-bd9e-11e8-923a-e1058f294f83.png)

**Comment by jasongrout** [comment]
Ah, that clears things up. So this *isn't* the message in this issue (the message at the top of this issue) - that was a red herring for me. We probably should have carried this discussion out on a new issue, with this explanation and screenshot, in the ipywidgets repo, rather than augmenting an already solved and closed issue.

But no matter. We're glad you asked for help. Do you see any javascript errors in the browser javascript console? Can you also try doing `jupyter lab build` just to make sure the extension got built into lab?

**Comment by Gabrielvon** [comment]
Sorry for the misleading. I just opened [a new repo](https://github.com/jupyter-widgets/ipywidgets/issues/2220) for my issues. 

Where can I find the browser javascript console? Actually, I am using a remote system maintained by my colleagues but they simply assign me a virtual system that I can do what I want. As the results, I don't know if I can see the prompt message about running jupyter lab.

And I ran `jupyter lab build` and it presented the following. Did I build it correctly?

![image](https://user-images.githubusercontent.com/8952869/45862424-3a961280-bda4-11e8-87fa-c708c9e6cccc.png)

![image](https://user-images.githubusercontent.com/8952869/45862430-41248a00-bda4-11e8-8753-eefcfc09b014.png)
